news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+
+What other rhythm games do you play (or have played), and do you think playing them helped you in becoming better at osu!catch?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+I used to play O2jam, DJMAX, EZ2AC, SDVX, and I actually do think playing them helped me a lot in becoming better at speed, precision and stamina for osu!catch.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+As a very approachable person to have interactions with, what do you think of osu!catch community as it is right now?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+First of all, I'm so honored and glad that I am considered that kind of person! :)
+
+I would say that most people in the community, including top players are chill and cool people, everyone is approachable and easy to have interactions with. I bet it will be easy to interact with us and melt into our community.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Do you have any advice that you'd like to give players that want to or just started playing osu!catch?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+I think it would be better to check for [YesMyDarknesss’s interview](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) because it was such a neat answer that there's not much to say for me. He's definitely a better fruit teacher than I am :)
```

Comment:
```suggestion
I think it would be better to check for [YesMyDarknesss’ interview](https://osu.ppy.sh/home/news/2024-05-28-the-followpoint-yesmydarknesss-the-master-of-osu-catch) because it was such a neat answer that there's not much to say for me. He's definitely a better fruit teacher than I am :)
```